loopd/daemon.go

```diff
@@ -175,7 +178,7 @@ func (d *Daemon) startWebServers() error {
 
 	// Next, start the gRPC server listening for HTTP/2 connections.
 	log.Infof("Starting gRPC listener")
-	d.grpcListener, err = d.listenerCfg.grpcListener()
+	d.grpcListener, err = d.listenerCfg.grpcListener(nil)
```

Comment:
If it crashes with `nil`, this isn't a working commit and possibly a sign that it should be squashed with another one. But non-blocking.